lib/themes/dosomething/paraneue_dosomething/js/reportback/Reportback.js

```diff
@@ -183,34 +182,40 @@ const Reportback = {
    * @param  {array} data  Array of Kudos objects from API.
    * @return {array}
    */
-  getReactions: function(data) {
-    let reactions = [];
-
-    forEach(this.reactions.approved, (term) => {
-      const termId = this.reactions.terms[term];
-
-      let reaction = {
-        kudosId: null,
-        term: term,
-        termId: termId,
-        total: 0,
-        userReacted: false,
-      };
+    getReactions: function(data) {
```

Comment:
Oh sorry, I also realized this looks like it got accidentally indented too! Can you fix this before we merge?